pkg/kosmosctl/floater/check.go

```diff
@@ -92,6 +94,7 @@ func NewCmdCheck() *cobra.Command {
 	flags.StringVar(&o.Port, "port", "8889", "Port used by floater.")
 	flags.IntVarP(&o.PodWaitTime, "pod-wait-time", "w", 30, "Time for wait pod(floater) launch.")
 	flags.StringVar(&o.Protocol, "protocol", string(TCP), "Protocol for the network problem.")
+	flags.IntVarP(&o.routinesMaxNum, "routines-max-number", "", 5, "Number of goroutines to use.")
```

Comment:
It seems to be a feasible method. Can a single test provide more detailed loss output?
Considering that excessive goroutines may cause a certain burden on the native network, if routines-max-number is set to 100 or more, how much traffic will the floater consume?

Reply:
ok, i will try it